sunpy/map/tests/test_mapbase.py

```diff
@@ -1073,16 +1074,25 @@ def test_contour(simple_map):
 def test_contour_units(simple_map):
     # Check that contouring with units works as intended
     simple_map.meta['bunit'] = 'm'
+
+    # Same units
     contours = simple_map.contour(1.5 * u.m)
     assert len(contours) == 1
 
+    # Different units, but convertible
     contours_cm = simple_map.contour(150 * u.cm)
     for c1, c2 in zip(contours, contours_cm):
         np.all(c1 == c2)
 
-    with pytest.raises(u.UnitsError, match='level must be an astropy quantity convertible to m'):
+    # Percentage
+    contours_percent = simple_map.contour(100 * u.percent)
+    contours_ref = simple_map.contour(np.max(simple_map.data) * simple_map.unit)
+    for c1, c2 in zip(contours_percent, contours_ref):
+        np.all(c1 == c2)
```

Comment:
Should this have an assert?
```suggestion
        assert np.all(c1 == c2)
```

Reply:
🙈 good catch!